patches/chrome-installer-util-google_update_settings.cc.patch

```diff
@@ -0,0 +1,36 @@
+diff --git a/chrome/installer/util/google_update_settings.cc b/chrome/installer/util/google_update_settings.cc
+index 42a2c0cc1b9576aa5f6c03e615e4b8450611831d..c2c8cc5e7d0bbdc6bf1b073ccfeec4c547497fff 100644
+--- a/chrome/installer/util/google_update_settings.cc
++++ b/chrome/installer/util/google_update_settings.cc
+@@ -36,7 +36,11 @@ using base::win::RegKey;
+ using installer::InstallationState;
+ 
+ const wchar_t GoogleUpdateSettings::kPoliciesKey[] =
++#if defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD)
++    L"SOFTWARE\\Policies\\BraveSoftware\\Update";
++#else
+     L"SOFTWARE\\Policies\\Google\\Update";
++#endif
+ const wchar_t GoogleUpdateSettings::kUpdatePolicyValue[] = L"UpdateDefault";
+ const wchar_t GoogleUpdateSettings::kDownloadPreferencePolicyValue[] =
+     L"DownloadPreference";
+@@ -48,6 +52,11 @@ const wchar_t GoogleUpdateSettings::kCheckPeriodOverrideMinutes[] =
+ const int GoogleUpdateSettings::kCheckPeriodOverrideMinutesMax =
+     60 * 24 * 7 * 6;
+ 
++
++#if defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD)
++#define GOOGLE_CHROME_BUILD
++#endif
++
+ const GoogleUpdateSettings::UpdatePolicy
+ GoogleUpdateSettings::kDefaultUpdatePolicy =
+ #if defined(GOOGLE_CHROME_BUILD)
+@@ -913,3 +922,7 @@ bool GoogleUpdateSettings::ReadExperimentLabels(
+ 
+   return result == ERROR_SUCCESS;
+ }
++
++#if defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD)
++#undef GOOGLE_CHROME_BUILD
++#endif
```

Comment:
I don't think you need to worry about this since you're only defining it in the single .cc file.

Reply:
This would be needed when we enable jumbo build later.

Reply:
Good point, I'll watch out for that in the future on reviews too.